core/dbt/contracts/graph/manifest.py

```diff
@@ -427,7 +427,11 @@ def add_node(self, node):
     # This should return a list of disabled nodes. It's different from
     # the other Lookup functions in that it returns full nodes, not just unique_ids
     def find(
-        self, search_name, package: Optional[PackageName], version: Optional[NodeVersion] = None
+        self,
+        search_name,
+        package: Optional[PackageName],
+        resource_types: Optional[List[NodeType]] = None,
```

Comment:
I'll move `resource_types` to be the last argument in case there are any existing order-dependent callsites that pass in `version` as the third argument.